tools/autograd/derivatives.yaml

```diff
@@ -951,6 +951,12 @@
 - name: sum.dim_IntList(Tensor self, int[1] dim, bool keepdim=False, *, ScalarType? dtype=None) -> Tensor
   self: sum_backward(grad, self.sizes(), dim, keepdim)
 
+- name: nansum(Tensor self, *, ScalarType? dtype=None) -> Tensor
+  self: grad.expand(self.sizes()) * self.isnan().logical_not()
```

Comment:
Shoudn't the `dtype` argument be used somewhere here? (and same for the formula below?

Reply:
Done.

Reply:
Looks good.
Could you add one more test to make sure it works fine?